simplified_galaxy_stable_container_creation.sh

```diff
@@ -1,114 +1,145 @@
            docker push $GALAXY_INIT_PHENO_TAG
        fi
 fi
 
-GALAXY_INIT_PHENO_FLAVOURED_TAG=$DOCKER_REPO$DOCKER_USER/galaxy-init-pheno-flavoured:$TAG
 
-DOCKERFILE_INIT_FLAVOUR=Dockerfile_init
 if [ -n $GALAXY_REPO ]
 then
-	echo "Making custom galaxy-init-pheno-flavoured:$TAG starting from galaxy-init-pheno:$TAG"
-	sed s+pcm32/galaxy-init-pheno$+$GALAXY_INIT_PHENO_TAG+ $DOCKERFILE_INIT_FLAVOUR > Dockerfile_init_tagged
-	DOCKERFILE_INIT_FLAVOUR=Dockerfile_init_tagged
+  echo "Making custom galaxy-init-pheno-flavoured:$TAG starting from galaxy-init-pheno:$TAG"
+  sed s+pcm32/galaxy-init-pheno$+$GALAXY_INIT_PHENO_TAG+ $DOCKERFILE_INIT_FLAVOUR > Dockerfile_init_tagged
+  DOCKERFILE_INIT_FLAVOUR=Dockerfile_init_tagged
 fi
 docker build $NO_CACHE -t $GALAXY_INIT_PHENO_FLAVOURED_TAG -f $DOCKERFILE_INIT_FLAVOUR .
-docker push $GALAXY_INIT_PHENO_FLAVOURED_TAG 
 
-GALAXY_WEB_K8S_TAG=$DOCKER_REPO$DOCKER_USER/galaxy-web-k8s:$TAG
+if [[ "${DOCKER_PUSH_ENABLED:-}" = "true" ]]; then
+  docker push $GALAXY_INIT_PHENO_FLAVOURED_TAG
+fi
+
 
-DOCKERFILE_WEB=Dockerfile
 if [ -n $GALAXY_REPO ]
 then
-	echo "Making custom galaxy-web-k8s:$TAG from $GALAXY_REPO at $GALAXY_RELEASE"
-	sed s+$GALAXY_BASE_FROM_TO_REPLACE+$GALAXY_BASE_PHENO_TAG+ docker-galaxy-stable/compose/galaxy-web/Dockerfile > docker-galaxy-stable/compose/galaxy-web/Dockerfile_web
-	DOCKERFILE_WEB=Dockerfile_web
+  echo "Making custom galaxy-web-k8s:$TAG from $GALAXY_REPO at $GALAXY_RELEASE"
+  sed s+$GALAXY_BASE_FROM_TO_REPLACE+$GALAXY_BASE_PHENO_TAG+ docker-galaxy-stable/compose/galaxy-web/Dockerfile > docker-galaxy-stable/compose/galaxy-web/Dockerfile_web
+  DOCKERFILE_WEB=Dockerfile_web
 fi
 docker build $NO_CACHE --build-arg GALAXY_ANSIBLE_TAGS=supervisor,startup,scripts,nginx,k8,k8s -t $GALAXY_WEB_K8S_TAG -f docker-galaxy-stable/compose/galaxy-web/$DOCKERFILE_WEB docker-galaxy-stable/compose/galaxy-web/
-docker push $GALAXY_WEB_K8S_TAG
+
+if [[ "${DOCKER_PUSH_ENABLED:-}" = "true" ]]; then
+  docker push $GALAXY_WEB_K8S_TAG
+fi
 
 # Build postgres
-POSTGRES_TAG=$DOCKER_REPO$DOCKER_USER/galaxy-postgres:$POSTGRES_VERSION"_for_"$GALAXY_VER_FOR_POSTGRES
 docker build -t $POSTGRES_TAG -f docker-galaxy-stable/compose/galaxy-postgres/Dockerfile docker-galaxy-stable/compose/galaxy-postgres/
-docker push $POSTGRES_TAG
+
+if [[ "${DOCKER_PUSH_ENABLED:-}" = "true" ]]; then
+  docker push $POSTGRES_TAG
+fi
 
 # Build proftpd
-PROFTPD_TAG=$DOCKER_REPO$DOCKER_USER/galaxy-proftpd:for_galaxy_v$GALAXY_VER_FOR_POSTGRES
 docker build -t $PROFTPD_TAG -f docker-galaxy-stable/compose/galaxy-proftpd/Dockerfile docker-galaxy-stable/compose/galaxy-proftpd/
-docker push $PROFTPD_TAG
+
+if [[ "${DOCKER_PUSH_ENABLED:-}" = "true" ]]; then
+  docker push $PROFTPD_TAG
+fi
 
 
 echo "Relevant containers:"
 echo "Web:          $GALAXY_WEB_K8S_TAG"
 echo "Init Flavour: $GALAXY_INIT_PHENO_FLAVOURED_TAG"
 echo "Postgres:     $POSTGRES_TAG"
 echo "Proftpd:      $PROFTPD_TAG"
+
+if [[ "${DOCKER_PUSH_ENABLED:-}" = "true" ]]; then
```

Comment:
Isn't this message misleading @ilveroluca ?

Reply:
No, I wouldn't say misleading.  It's downright wrong!  :sweat_smile:.  Sorry.  I'll fix that right away on a new PR